External/FEXCore/Source/Interface/IR/IR.json

```diff
@@ -75,7 +75,13 @@
     "constexpr FEXCore::IR::BreakReason Break_Interrupt3         {2}",
     "constexpr FEXCore::IR::BreakReason Break_Halt               {3}",
     "constexpr FEXCore::IR::BreakReason Break_Overflow           {4}",
-    "constexpr FEXCore::IR::BreakReason Break_InvalidInstruction {5}"
+    "constexpr FEXCore::IR::BreakReason Break_InvalidInstruction {5}",
+
+    "constexpr uint8_t SYSCALL_FLAG_DEFAULT            {0}",
```

Comment:
APIs with untyped "flag" parameters are difficult to understand and follow, so these would be better suited as an `enum class`.

 For the composition operators, FWIW Mesa has a set of useful macros to autogenerate them if we don't already have something equivalent: https://gitlab.freedesktop.org/mesa/mesa/-/blob/main/src/util/enum_operators.h